twin/voicemail-record.php

```diff
@@ -30,18 +30,16 @@
 
 if (!$url) {
 	// no voicemail!
-	$response->say('An error occurred - your voicemail was not received. Goodbye.',
+	$response->sms_playOrSay('An error occurred - your voicemail was not received. Goodbye.',
```

Comment:
This should be "sms_playOrSay($response," instead of "$response->sms_playOrSay("